udp_to_mcu:
  mcu_ip: 192.168.123.110
  controller_port: 8881

udp_to_sdk:
  sdk_ip: 127.0.0.1
  controller_port: 8072

collision:
  open: false
  torque_threhold: 10.0
  payload: 0.03
